product-service/src/handlers/get-product-list/handler.spec.ts

```diff
@@ -7,11 +7,9 @@ const productsMock: ProductDTO[] = [
     { id: 'id-1', title: 'Product 1', description: 'Description 1', price: 1, count: 1 },
     { id: 'id-2', title: 'Product 2', description: 'Description 2', price: 2, count: 2 }
 ];
-jest.mock('@lib/services/product.service.ts', () => ({
-    default: function () {
-        return { getAll: () => Promise.resolve(productsMock) }
-    }
-}));
+jest.mock('@lib/services/product.service.ts', () => function() {
+    return { getAll: () => Promise.resolve(productsMock) }
```

Comment:
Commonly jest throws an error in case when we mock module and try to put inside mock variable from outer scope.   Do you have such issue?

Reply:
No issues. It works
<img width="428" alt="image" src="https://user-images.githubusercontent.com/102066028/227550275-861ae0f8-2f4d-41ea-8094-0304bb018183.png">